Cargo.lock

```diff
@@ -2859,13 +2912,22 @@ dependencies = [
  "bitflags",
 ]
 
+[[package]]
+name = "redox_syscall"
+version = "0.3.5"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "567664f262709473930a4bf9e51bf2ebf3348f2e748ccc50dea20646858f8f29"
+dependencies = [
+ "bitflags",
+]
+
 [[package]]
 name = "regex"
-version = "1.7.1"
+version = "1.8.1"
```

Comment:
Ok, so im wondering if we want to continue having this test, why should we test that a regex is failing at a certain point?

Reply:
The test has the following comment:

> The data won't be scrubbed here, since the regex cannot be compiled.

Testing a sensitive field is not being scrubbed when a regex cannot be compiled seems a legit use case to me. Why do you think it's not?

Reply:
well we don't want the regex to fail, and now the new update to regex makes it not fail, so id say thats a good thing, we can't keep the test as it is now unless we try to stop regex from being upgraded